branding/brand.css

```diff
@@ -132,3 +132,12 @@ over) by other installed brands.
     bottom: 25px;
     right: 25px;
 }
+
```

Comment:
this file is actually reserved for things that are copied from the admin portal branding file. (We can't put things in here because they'll be overwritten eventually.)

you should be able to create a new style.css in /src and import it:
import style from './style.css'